rogerokello/weconnect-practice
# Branch containing site pages and class diagram

**What does this pull request do?**
Adds template pages for a user to register themselves, sign in, create a business profile, edit a business profile, list businesses and search for them, review a business and view business reviews.

**Where should the reviewer start?**
If you want to check out the pages click on https://rogerokello.github.io/weconnect-practice/ to view the templates in action.

**How should this be manually tested?**
Since these are only templates and not working versions, there are currently no tests. However, you may click on links and see if they take you to the relevant pages.

**Screenshots**
See the app live at https://rogerokello.github.io/weconnect-practice/
